pygeo/parameterization/DVGeo.py

```diff
@@ -2446,6 +2498,38 @@ def addVariablesPyOpt(
                 ("spanwiselocalVars", self.DV_listSpanwiseLocal),
             ]
         )
+
+        # we add the composite DVs, and construct linear constraints that replace the existing bounds
+        # then we simply return without adding any of the other DVs
+        if self.useComposite:
+            dv = self.DVComposite
+            optProb.addVarGroup(dv.name, dv.nVal, "c", value=dv.value, lower=dv.lower, upper=dv.upper, scale=dv.scale)
+
+            # add the linear DV constraints that replace the existing bounds!
+            # Note that we assume all DVs are added here, i.e. no ignoreVars or any of the vars = False
```

Comment:
Is this a limitation? I can't picture a production case where we use `ignoreVars`, but maybe we can add a warning message to stay on the safe side?

Reply:
Yeah, I can add a warning if ignoreVars is passed in.